main/res/values/strings.xml

```diff
@@ -890,6 +890,9 @@
     <string name="init_backup_history_length_min">Only recent</string>
     <string name="init_backup_history_length_max">Keep all</string>
     <string name="init_backup_history_delete_warning">You have saved more backups than your configured maximum value allows. Do you really want to remove all excess backups?</string>
+    <string name="init_backup_automatic">Automatic backup reminder</string>
+    <string name="init_backup_automatic_summary">Reminds you every x days to create an automatic backup (in addition to any manually created backups), 0=off</string>
+    <string name="init_backup_automatic_reminder">Shall I create an automatic backup for you now? This will only take a short while.</string>
```

Comment:
IMHO the "Shall I" Form should only be used in the installation wizard, but nowhere else...

Reply:
I would also stay with something more formal here.

Reply:
Aligned the text to the map/routing tile reminders: "Do you want to create an automatic backup? ..."